src/types.ts

```diff
@@ -13,10 +13,16 @@ export type IMethodCall = [
   string
 ];
 
-export interface IGetArguments<Properties> {
+export interface IGetArguments {
   accountId: string;
   ids: string[] | null;
-  properties?: Properties[];
+  properties?: string[];
```

Comment:
I wanted to use the type argument to tell `properties` is a list of keys of `Properties`, but I made an error defining it, it should have been `(keyof Properties)[]` instead of `Properties[]`, that is probably why you removed it?

Reply:
I think so, i shall repair it